src/kleinanzeigen_bot/__init__.py

```diff
@@ -3,30 +3,74 @@
 SPDX-License-Identifier: AGPL-3.0-or-later
 SPDX-ArtifactOfProjectHomePage: https://github.com/Second-Hand-Friends/kleinanzeigen-bot/
 """
-import asyncio, atexit, copy, importlib.metadata, json, logging, os, re, signal, shutil, sys, textwrap, time
```

Comment:
I know that single-line imports violate PEP 8, but I disagree with blindly following this rule, especially since no clear rationale is provided in the spec except that doing so "is wrong". One can also argue that expanding the import section solely to comply with PEP 8 conflicts with PEP 20, the Zen of Python, which emphasizes that "Simple is better than complex" and "Sparse is better than dense."